css/at-rules/media.json

```
@@ -1303,17 +1303,22 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/prefers-reduced-transparency",
             "spec_url": "https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-transparency",
             "support": {
-              "chrome": {
-                "version_added": "preview",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#enable-experimental-web-platform-features",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "impl_url": "https://crbug.com/1424879"
-              },
+              "chrome": [
+                {
+                  "version_added": "118"
+                },
+                {
+                  "version_added": "117",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "#enable-experimental-web-platform-features",
+                      "value_to_set": "Enabled"
+                    }
+                  ],
+                  "impl_url": "https://crbug.com/1424879"
+                }
+              ],
```

When the flag is no longer relevant, we prefer to remove it from BCD:

```suggestion
              "chrome": {
                "version_added": "118"
              }
```

---

Ah that makes sense will do that.